code/script.js

```diff
@@ -1,51 +1,270 @@
+    botReply(`More history for the people: The pineapple spread throughout South America, the Caribbean, Central America and Mexico, where it was cultivated by the Mayas and the Aztecs. Columbus came across the pineapple in 1493 and took it back to Europe.`), 1500)
+
+    setTimeout ( () =>
+    botReply ('Are you ready to order your pineapple?'), 1000)
+
+  wrapper.innerHTML =`
+  <button id="order" > Yes </button>
+ 
+  `;
+  //egen function för den med slutklämmen
+
+  document.getElementById('order').addEventListener('click', () => answerThree("order"))
+
+} else if ( type === 'funFactTwo' ){
+  console.log('vid fun fact nr 2')
+
+  setTimeout ( () =>
+  userReply ('More pineapples to the people'), 1000)
+  setTimeout ( () =>
+  botReply ('Pineapples were such a status symbol in 18th century England that you could rent one for the evening to take to a party.'), 1000)
+
+
+  setTimeout ( () =>
+  botReply ('Are you ready to order your pineapple?'), 1500)
+
+  wrapper.innerHTML =`
+<button id="order" > Yes </button>
+
+`;
+
+
+document.getElementById('order').addEventListener('click', () => answerThree("order"))
+
+}}
+
+const answerThree = (fin) => {
+
+ if (fin === 'order') {
+
+  setTimeout ( () =>
+  userReply ("Of course I want to order! finaly I get my own Pineapple"), 1000)
+   
+  setTimeout ( () =>
+  botReply ('Perfect choose your size here.'), 1000)
+
+    //fixa en knapp för de olika storlekarna sen en form för mail
+wrapper. innerHTML = `
+
+<label for="select your fruit"></label>
+
+<select id="select">
+    <option value="" selected disabled>--What kind of pineapple do you want=?--</option>
+    <option value="Gianormous">Gianormous</option>
+    <option value="Medium Pineapple">Not to big</option>
+    <option value="Small Pineapple">Small</option>
+    <option value=" Piña colada">Piña colada</option>
+    <option value="frozen pineapple">Frozen </option>
+    <option value=" at a beach">At a beach</option>
+</select>`
+   }
+const select = document.getElementById('select')
+select.addEventListener('change', () => endForm(select.value))
+
+
+
+
+}
+
+    
+    // This wont work?
+  const endForm = (pineapple) => {
+    setTimeout ( () =>
+    botReply (`Yey thanx for orderinge a ${pineapple } write your name and e mail and we will send you information on where to get your happy pineapple`), 1000)
+
+wrapper.innerHTML=
+`<form>
+
+<input id="email" name="email" type="email" required placeholder= "e-mail">
+
+
+<button id="button" > Send </button>
+</form>`
+
+
+document.getElementById('email').addEventListener('click', () => thankYou())
+document.getElementById('button').addEventListener('click', () => thankYou())
+} 
+
+//these wont work!
+const thankYou =() => {
+  setTimeout ( () =>
+  botReply ("Pineapples are international symbols of welcome and are a symbolic way of saying “you are perfect” when presented to someone."), 1000)
+setTimeout ( () =>
+botReply ('Thank you for ordering hope to see you again!'), 1000)
```

Comment:
My guess for it not working is because your are not invoking it after the order is sent. You can fix this! Judging from all the other code above you know how to solve this!